glossary.dd

```diff
@@ -81,6 +81,14 @@ $(DL
 	maximize the portability of code.
 	)
 
+	$(DT $(LNAME2 lvalue, lvalue))
+	$(DD A lvalue is the counterpart to a rvalue. A lvalue is an abstract term
```

Comment:
Isn't it "An lvalue"? (Here we go again...)

The "X is the counterpart of Y" part should probably be at the end, to avoid inviting people to look up the other term only to get a reference to the first one.